hswindow/hsdialog/hsprojectpropertiesdialog/projectpropertiesdialog.go

```diff
@@ -157,8 +157,10 @@ func (p *ProjectPropertiesDialog) Build() {
 									}),
 									g.ImageButton(p.downIconTexture).Size(imgBtnW, imgBtnH).OnClick(func() {
 										if currentIdx < len(p.project.AuxiliaryMPQs)-1 {
-											p.project.AuxiliaryMPQs[currentIdx] = p.project.AuxiliaryMPQs[currentIdx+1]
-											p.project.AuxiliaryMPQs[currentIdx+1] = p.project.AuxiliaryMPQs[currentIdx]
+											p.project.AuxiliaryMPQs[currentIdx],
+												p.project.AuxiliaryMPQs[currentIdx+1] =
+												p.project.AuxiliaryMPQs[currentIdx+1],
+												p.project.AuxiliaryMPQs[currentIdx]
```

Comment:
good catch

Reply:
generally it was linter's catch :grinning: 
my was like that
```golang
prev := sth[idx]
sth[idx+1]=sth[idx]
sth[idx] = prev
```